pkg/types/types.go

```diff
@@ -76,3 +78,36 @@ type MercuryProvider interface {
 	OnchainConfigCodec() mercury.OnchainConfigCodec
 	ContractTransmitter() mercury.Transmitter
 }
+
+type FunctionsProvider interface {
+	PluginProvider
+	LogPollerWrapper() LogPollerWrapper
+	NodeSignerKey() *ecdsa.PrivateKey
+}
+
+type Address [20]byte
```

Comment:
Are you able to use the `libocr.Account` type instead? https://github.com/smartcontractkit/libocr/blob/master/offchainreporting2plus/types/types.go#L297-L298

Reply:
boo, I wanted a fixed size byte array but ok...

Reply:
This is sort of a chain agnostic gray area - we must avoid chain-specific types/constraints, but we can have chain-specific data and encodings, just as long as we don't make any assumptions like by trying to parse them. Broadly speaking, we can pass data around from the config and relayer, but can't really interact with it other than simple things like raw comparisons.